python/lsst/ctrl/bps/cli/script/prepare.py

```diff
@@ -0,0 +1,52 @@
+# This file is part of ctrl_bps.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+from lsst.daf.butler.cli.cliLog import CliLog
+from ...bps_config import BpsConfig
+from ...submit import BPS_SEARCH_ORDER, create_submission
+
+
+def _prepare(config_file, log_level, **kwargs):
+    """Create a workflow for a specific workflow management system.
+
+    Parameters
+    ----------
+    config_file : `str`
+        Name of the configuration file.
+    log_level : `str`
+        Per-component logging levels, each item in the list is a tuple
+        (component, level), `component` is a logger name or an empty string
```

Comment:
I think to work properly with sphinx, non-keyword monospace words must be quoted in double backticks, because sphinx tries to make reference out of single backtick'ed vars. So for example this section should read like so:
```
    log_level : `str`
         Per-component logging levels, each item in the list is a tuple
         (component, level), ``component`` is a logger name or an empty string
```
(It's very possible you copied this docstring from my code; and if so you have my apologies, I didn't know about this and have been fixing as I go.

Reply:
Yes, I shamelessly copy-pasted this description from your code. I'll fix the docstrings in my code as you described.